attributes/default.rb

```diff
@@ -4,6 +4,7 @@
 default.loggly.tags = []
 
 default.loggly.log_files = []
+default.loggly.apps = {}
```

Comment:
I don't think "apps" idiomatic within either loggly or rsyslog.  I made it up, and we would have to define it in the docs.  Something like "file_groups" would be more explicit, but it was just more cumbersome.  It may be better to be explicit, however, to avoid conflicting with the built-in "app-name" concept from rsyslog.  Let me know which way you prefer.